amostra/client/commands.py

```diff
@@ -337,3 +373,114 @@ def get_schema(self):
         r.raise_for_status()
         return ujson.loads(r.text)
 
+
+class LocalSampleReference:
+    """Handle sample information locally via json files"""
+    def __init__(self, top_dir=conf.local_conn_config['top']):
+        self.top_dir = top_dir
+        try:        
+            with open(self._samp_fname, 'r') as fp:
+                tmp = ujson.load(fp)
+        except FileNotFoundError:
+            tmp = []
+        try:
+            self.sample_list = tmp if tmp else []
+        except FileNotFoundError:
+            self.sample_list = []
+        
+    def create(self,name=None, time=None, uid=None, container=None,
+               **kwargs):
+        payload = dict(uid=uid if uid else str(uuid4()),
+                   name=name, time=time if time else ttime.time(),
+                   container=container if container else 'NULL',
+                   **kwargs)
+        self.sample_list.append(payload)
+        with open(self._samp_fname, 'w+') as fp:
+            ujson.dump(self.sample_list, fp)
+        return payload
+        
+    @property
+    def _samp_fname(self):
+        return  expanduser(self.top_dir + '/samples.json')
+    
+    def update(self):
+        with open(self._samp_fname, 'a+') as fp:
+            local_payload = ujson.load(fp)
+
+    def find(self):
+        with open(self._samp_fname, 'r') as fp:
+            print(fp)
+            local_payload = ujson.load(fp)
+        print(local_payload, type(local_payload))
+        
+        
+class LocalRequestReference:
+    def __init__(self, top_dir=conf.local_conn_config['top']):
+        self.top_dir = top_dir        
+        try:        
```

Comment:
Seems like you duplicate this json loading logic in a few places, might want to break it out into a separate function